todo_app/app.py

```diff
@@ -1,11 +1,43 @@
 from flask import Flask
-
+from flask.templating import render_template
 from todo_app.flask_config import Config
+from todo_app.data import session_items
+from flask import request
+
+
+import logging
+log = logging.getLogger('werkzeug')
+log.setLevel(logging.DEBUG)
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    todos = session_items.get_items()
+    
+    return render_template('index.html', todos=todos)
+
+@app.route('/modify/<id>')
+def modify_todo(id):
+    todo = session_items.get_item(id)
+    return render_template('modify.html', todo=todo)
+
+@app.route('/add_todo', methods=['POST'])
+def add_todo():
+    session_items.add_item(request.form['title'])
+    return index()
```

Comment:
Note that this works, but it means the user sees a direct page response to their POST request & their browser can get confused in this case with things like:
* If you try refreshing the page after creating a new to-do, you'll find that your browser tries to create another one
* If you bookmarked the page after creating a new to-do, the bookmark would fail when you return to it (since it would execute a GET request to this path)

Instead we should _redirect_ them once the function has succeeded - that way the browser knows that the loaded page is from the index route. Flask provides a handy redirect function for this so we can do so with something like:
```python
return redirect(url_for('index'))
```